test/snippet/utility/char_operations/char_predicate.cpp

```diff
@@ -4,55 +4,55 @@
 int main()
 {
     //! [is_eof]
-    seqan3::is_eof(EOF); // returns true
-    seqan3::is_eof('C'); // returns false
+    static_assert(seqan3::is_eof(EOF));  // returns true
```

Comment:
why the extra static_assert? And if so, the comment is misleading. It should now state `asserts` or `fine/does not assert` sine it doesn't return anything